packages/runtime-common/realm.ts

```diff
@@ -234,8 +234,11 @@ export class Realm {
   #router: Router;
   #deferStartup: boolean;
   #useTestingDomain = false;
+  // TODO remove this after we serve transpiled js from index
   #transpileCache = new Map<string, string>();
   #log = logger('realm');
+  #perfLog = logger('perf');
+  #startTime = Date.now();
```

Comment:
note that i'm not using `performance.now()` in this module because it's not isomorphic. Specifically in nodejs you need to import `performance` from `perf_hooks` and in the browser this hangs off of the `window`. This dichotomy makes it difficult to use `performance.now()`, so on the server we just use `Date.now()`.